apps/dashboard/src/modules/admin/components/users/AdminUserInfoCard.vue

```diff
@@ -0,0 +1,64 @@
+<template>
+  <FormCard :header="$t('userDetails.userInformation')"
+            @update:modelValue="edit = $event"
+            @save="formSubmit"
+            :enableEdit="true"
+  >
+      <div class="flex flex-column justify-content-between gap-2">
+        <UserEditForm :user="props.user" :form="form" :edit="edit" @update:edit="edit = $event"/>
+      </div>
+  </FormCard>
+</template>
+
+<script setup lang="ts">
+import FormCard from "@/components/FormCard.vue";
+import { onMounted, type PropType, ref, watch } from "vue";
+import type { UserResponse } from "@sudosos/sudosos-client";
+import { schemaToForm } from "@/utils/formUtils";
+import { updateUserDetailsObject, userTypes } from "@/utils/validation-schema";
+import UserEditForm from "@/modules/admin/components/users/forms/UserEditForm.vue";
+
+const props = defineProps({
+  user: {
+    type: Object as PropType<UserResponse>,
+    required: true
+  }
+});
+
+const edit = ref(false);
+
+const form = schemaToForm(updateUserDetailsObject);
+
+const formSubmit = () => {
+  form.submit();
+};
+
+const updateFieldValues = (p: UserResponse) => {
+  if (!p) return;
+  const values = {
+    firstName: p.firstName,
+    lastName: p.lastName,
+    email: p.email,
+    nickname: p.nickname,
+    isActive: p.active,
+    userType: userTypes.value.find(ut => ut.name === p.type)?.value || undefined,
+    ofAge: p.ofAge,
+    canGoIntoDebt: p.canGoIntoDebt,
+  };
+  form.context.resetForm({ values });
```

Comment:
prefer `{ ....p,` and only specify things you change, so in this case
```ts
{
  ...p,
  userType: userTypes.value.find(ut => ut.name === p.type)?.value || undefined,
}
```

Reply:
Fixed in latest commit